Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx

```diff
@@ -31,18 +35,22 @@ const HomeContests = () => {
     );
 
     return (
-        <>
-            <ContestsList
-              title="Active"
-              contests={activeContests}
-              contestStatus={getContestStatusIndex(ContestStatus.Active)}
-            />
-            <ContestsList
-              title="Past"
-              contests={pastContests}
-              contestStatus={getContestStatusIndex(ContestStatus.Past)}
-            />
-        </>
+        isLoaded
+            ? (
+                <>
+                    <ContestsList
+                      title="Active"
+                      contests={activeContests}
+                      contestStatus={getContestStatusIndex(ContestStatus.Active)}
+                    />
+                    <ContestsList
+                      title="Past"
+                      contests={pastContests}
+                      contestStatus={getContestStatusIndex(ContestStatus.Past)}
```

Comment:
Get status index internally in the ContestsList component. `ContestStatus.Past` can be used both for the `title `and `contestStatus ` fields